nuxeo-web-ui-ftest/scripts/specs-processor.js

```diff
@@ -4,9 +4,9 @@ const fs = require('fs');
 module.exports = (argv) => {
   let features = [];
   const args = require('minimist')(argv.slice(2));
-  if (args.cucumberOpts && args.cucumberOpts.tags) {
+  if (args.cucumberOpts && args.cucumberOpts.tagExpression) {
```

Comment:
We must keep in mind that tagExpressions support operators like `and` and `or`. This is ok here because we're just using one tag at a time and it works just like the previons `tags` parameter. But we should create a ticket to support expressions and perhaps leave a comment here stating that this will not work for tag expressions (just for tags).

Reply:
Ticket created here: https://jira.nuxeo.com/browse/NXP-26660